snap/info.go

```diff
@@ -1007,6 +1010,8 @@ func ReadInfo(name string, si *SideInfo) (*Info, error) {
 		return nil, &invalidMetaError{Snap: name, Revision: si.Revision, Msg: err.Error()}
 	}
 
+	bindImplicitHooks(info)
```

Comment:
Unrelated to this branch. I expanded the code of `ReadInfo` and `ReadInfoFromSnapFile` and noticed we don't validate in the former function, only the latter function does. Hmmm...

Reply:
that's intended I think, ReadInfo is used only on already installed snaps, I'm sure there are corner cases related to "try" tough